Extensions/data.md

```diff
@@ -0,0 +1,99 @@
+# Data
+
+## Table definition
+
+A table definition defines each column as `properties`, rows as `values`, optional filters as `filters` and an optional pagination strategy as `pagination`
+
+```json
+{
+    "data": {
+        "filter": { "filter-name": { "$type": "..." } },
+        "pagination": { "$type": "..." },
+        "properties": { "property-name": {} },
+        "values": [ { "property-name": { "value": "..." } } ]
+    }
+}
+```
+
+## Values
+
+```json
+[
+    { "value": "my simple value, displayed as the label" }
+    { "value": "https://example.com", "renderAs": { "$type": "link", "label": { "en": "Example" } } }
+]
+```
+
+## Values renderAs
+
+Specify how the value should be presented. The value will be presented as-is if not specified.
+
+- link
+
+### link
+
+```json
+{
+    "value": "link href", // Value becomes the href
+    "renderAs": {
+        "$type": "link",
+        "label": { "en": "My label" }
+    }
+}
+```
+
+## Filters
+
+A filter should have its `value` specified in the case of a pre-filtered data set.
+
+- input

```

Comment:
@antur84 should we change this to text since that's what's in release 10.66?

Reply:
The docs should match what's released.